### ISSUE TITLE
Team alerts input form activation

### ISSUE THREAD

**Issue by j-zimnowoda** [redkubes/otomi-core#1497]
**Is your feature request related to a problem? Please describe.** 
Team alerts should be only enabled if team alert manager is enabled.

**Comment by ElderMatt** [comment]
@j-zimnowoda I was not able to replicate. Am I missing something? 
[Private Zenhub Video](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBODVxQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--c2095886aa405618b849791d2c3aa91590b9a44b/Screen%20Recording%202024-04-17%20at%2011.59.31.mov)

**Comment by j-zimnowoda** [comment]
yes, from the screencast alert-manager is enabled in team settings, still alerts settings is not.
Take a look at team apps.
